test/ui-test/testSuites/suite_status/shared/steps/steps.py

```diff
@@ -14,13 +14,20 @@
 # *          with a pattern which is matched against the steps being executed.
 # *****************************************************************************
 from common.Common import *
+from utils.FileManager import *
 import time
 
+
 @When("the user restarts the app")
 def step(context):
     waitFor(lambda: currentApplicationContext().detach(), 500)
     time.sleep(5)
     startApplication("nim_status_client")
+    
+
+@When("the app data folder is erased")
+def step(context):
+    erase_directory("../../../../../Status/data")
```

Comment:
We should avoid relative paths like this. Maybe we should create an environment variable with the root path and then build the paths from there.